tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html

```diff
@@ -13,6 +13,9 @@
 
         <div class="footer-cta__button-wrapper">
             <a href="{{ contact_link }}" class="button footer-cta__button">{{ contact_action }}</a>
+
+            <p class="footer-cta__email-wrapper"><a class="footer-cta__email" href="mailto:{{ contact_email }}">{{ contact_email }}</a></p>
```

Comment:
Question: Do we need to wrap this in a conditional as the field in Wagtail that populates it isn't required?

Reply:
good point will fix that.